packages/design-system/src/components/RichRadioButton/RichRadioButton.utils.ts

```diff
@@ -0,0 +1,25 @@
+import { Tag, TagBeta, TagDestructive, TagInformation, TagSuccess, TagWarning } from '../Tag';
+import { TagVariant } from '../Tag/Tag';
+import { RichRadioButtonTag } from './RichRadioButton.types';
+
+export const getTagVariant = (tag: RichRadioButtonTag) => {
+	switch (tag.variant) {
+		case TagVariant.beta:
```

Comment:
why aren't you using the same naming than the Tag's one ? it could prevent this conversion step by simply doing `Tag[tag.variant] || Tag` or something like that, wdyt ?

Reply:
Removed this function as the Tag component has a prop to pass the variant.

The array of tags doesn't really have a strong type. Anything could be passed into that array